Src/CSharpier/CommandLineFormatter.cs

```diff
@@ -99,20 +93,52 @@ CancellationToken cancellationToken
                     ignoreFile,
                     result
                 );
+            }
 
-                await commandLineFormatter.FormatFiles(cancellationToken);
+            if (commandLineOptions.StandardInFileContents != null)
+            {
+                var path = commandLineOptions.DirectoryOrFilePaths[0];
+
+                var commandLineFormatter = await CreateFormatter(path);
+                if (commandLineFormatter == null)
+                {
+                    return 1;
+                }
+
+                await commandLineFormatter.FormatFile(
+                    commandLineOptions.StandardInFileContents,
+                    path,
+                    Encoding.UTF8,
+                    cancellationToken
+                );
+            }
+            else
+            {
+                foreach (var path in commandLineOptions.DirectoryOrFilePaths)
+                {
+                    var commandLineFormatter = await CreateFormatter(path);
+                    if (commandLineFormatter == null)
+                    {
+                        return 1;
+                    }
+
+                    await commandLineFormatter.FormatFiles(cancellationToken);
+                }
             }
 
             result.ElapsedMilliseconds = stopwatch.ElapsedMilliseconds;
-            ResultPrinter.PrintResults(result, console, commandLineOptions);
+            if (!commandLineOptions.ShouldWriteStandardOut)
+            {
+                ResultPrinter.PrintResults(result, console, commandLineOptions);
```

Comment:
Might still be worth writing this out to StdErr.

Reply:
If I'm reading up on this correctly, I think that would mean all of the results/error information could be written to stderr and that wouldn't interfere with the formatted file that is written to stdout.
Which would definitely be nice because when I was working on the change below CSharpier wasn't giving me any information and not formatting the text I was trying to pipe to it.